packages/big-design/src/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -40,146 +45,148 @@ const excludeIconProps = ({
   ...actionProps
 }: ButtonProps & Pick<ButtonGroupAction, 'text'>): ButtonGroupAction => actionProps;
 
-export const ButtonGroup: React.FC<ButtonGroupProps> = memo(({ actions, ...wrapperProps }) => {
-  const parentRef = createRef<HTMLDivElement>();
-  const dropdownRef = createRef<HTMLDivElement>();
-  const [isMenuVisible, setIsMenuVisible] = useState(false);
-  const [actionsState, setActionsState] = useState<ActionsState[]>([]);
-
-  useEffect(() => {
-    setActionsState(
-      actions.map((action) => ({
-        isVisible: true,
-        action: excludeIconProps(action),
-        ref: createRef<HTMLDivElement>(),
-      })),
-    );
-  }, [actions]);
+export const ButtonGroup: React.FC<ButtonGroupProps> = memo(
+  ({ actions, localization, ...wrapperProps }) => {
```

Comment:
Can we define a default localization object instead of adding the translations adhoc?

Something like:

```tsx
const defaultLocalization: Localization {
  more: 'more',
};

// ...

export const ButtonGroup: React.FC<ButtonGroupProps> = memo(
  ({ actions, localization = defaultLocalization, ...wrapperProps }) => {

// ...
      iconOnly={<MoreHorizIcon title={localization.more} />}
```

Reply:
I had it this way at some point and reverted to what I have now 😅 . In other words, I agree we this!